README.md

```diff
@@ -1,2 +1,49 @@
-# Spiral-Tech-Project
-The module one capstone project at microverse.
+#Tech in Africa (TiA Summit) Webpage
+
+# Description of the project.
+
+The project is a webpage design personalised version of the module one capstone project guide. It has been designed for a non-real summit tagged the Tech in Africa (TiA) hosted to highlight the achievemnts and growth of young African tech community through innovation. The summit is to among other things create avenue for tech community to interact and showcase latest tech trends and innovation in Africa's tech community as well as the challenges.
+
+Built With:
+-HTML
+
+- CSS
+
+- ESLint
+
+#Live Demo
+[Live Demo Link] (https://gids-dadi.github.io/The-portfolio-project/)
```

Comment:
- [ ] Please make sure that the app is deployed on github pages and the link is updated in the README. At the moment, this is a link to your portfolio.